build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
 }
 
 plugins {
-    id "org.sonarqube" version "3.5.0.2730"
+    id "org.sonarqube" version "4.0.0.2929"
```

Comment:
Sonarqube version shouldnt be changed, specially a major bump up. Will messup with our CICD for now

Reply:
That came from https://github.com/mparticle-integrations/mparticle-android-integration-appboy/pull/123/commits/e040f5351eba1c08a12f22a264501d089511c180 and I'm not sure why it even got in there. I'll revert it.